ci/azure/windows.yml

```diff
@@ -23,32 +23,33 @@ jobs:
         Write-Host "##vso[task.prependpath]$env:CONDA\Scripts"
         Write-Host "##vso[task.prependpath]$HOME/miniconda3/bin"
       displayName: 'Add conda to PATH'
+
     - script: conda update -q -n base conda
       displayName: 'Update conda'
+
     - bash: |
         conda env create -q --file ci\\deps\\azure-windows-$(CONDA_PY).yaml
       displayName: 'Create anaconda environment'
+
     - bash: |
         source activate pandas-dev
         conda list
         ci\\incremental\\build.cmd
       displayName: 'Build'
+
```

Comment:
#30393 should have gotten rid of the build.cmd on L37.  this isnt going to re-introduce it by accident?

Reply:
Apologies just merged master